### PR TITLE
[15.0][MIG] purchase_partner_selectable_option: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT36511

### CODE REVIEW

**Comment by cesar-tecnativa** [comment]
@victoralmau @pedrobaeza please review

File: purchase_partner_selectable_option/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
+
+#. Go to **Contacts > Contacts**
+#. Create a record and uncheck the "Selectable in orders" box on the Purchase part of the "Sales & Purchases" tab.
```
Comment (sswapnesh) [inline purchase_partner_selectable_option/readme/USAGE.rst]:
```suggestion
#. Create a record and uncheck the "Selectable in purchase orders" box on the Purchase part of the "Sales & Purchases" tab.
```


and also we should not use string="Selectable in orders" on partner form

**Comment by cesar-tecnativa** [comment]
@sswapnesh everthing was done

**Review by sswapnesh — APPROVED** [review]

File: purchase_partner_selectable_option/readme/USAGE.rst [hunk 2]
```diff
@@ -2,5 +2,4 @@ To use this module, you need to:
 
 #. Go to **Contacts > Contacts**
 #. Create a record and uncheck the "Selectable in orders" box on the Purchase part of the "Sales & Purchases" tab.
-#. Go to **Purchase > Orders > Request for Quotations**
-#. Create a record and the previously created partner will not appear in the Vendor field.
+#. Create a record and uncheck the "Selectable in purchase orders" box on the Purchase part of the "Sales & Purchases" tab.
```
Comment (pedrobaeza) [inline purchase_partner_selectable_option/readme/USAGE.rst]: This is not correct. You must do the change on the other sentence, not to remove these ones and repeat the previous sentence with this change. Even more, I would put the first 2 steps in CONFIGURE.rst

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**